install_config/aggregate_logging.adoc

```diff
@@ -1468,6 +1468,158 @@ have the same multi-tenant capabilities and your data will not be restricted by
 user access to a particular project.
 ====
 
+[[sending-logs-to-external-rsyslog]]
+== Sending Logs to an External Syslog Server
+
+Use the `fluent-plugin-remote-syslog` plug-in on the host to send logs to an
+external syslog server. 
+
+Set environment variables in the `logging-fluentd` or `logging-mux` deployment
+configurations:
+
+[source,yaml]
+----
+- name: REMOTE_SYSLOG_HOST <1>
+  value: host1
+- name: REMOTE_SYSLOG_HOST_BACKUP
+  value: host2
+- name: REMOTE_SYSLOG_PORT_BACKUP
+  value: 5555
+----
+<1> The desired remote syslog host. Required for each host.
+
+The above will build two destinations. The syslog server on `host1` will be
+receiving messages on the default port of `514`, while `host2` will be receiving
+the same messages on port `5555`.
+
+Alternatively, you can configure your own custom `fluent.conf` in the
+`logging-fluentd` or `logging-mux` ConfigMaps.
+
+[NOTE]
+====
+Any logs on the external rsyslog server will be raw journald output, and not
+formatted similar to the OpenShift Fluentd output. This means it will not be
+corrolatable and will have no OpenShift or Kubernetes metadata.
+====
```

Comment:
This is not true when using the fluentd syslog plugin.  When using this plugin, you get the records in the same format that is used to store them in the aggregated logging Elasticsearch, fully annotated with Kubernetes metadata.

This is only true if you use rsyslog to gather the logs, which is a completely different topic.

Reply:
Sure. Thanks @richm . I deleted the note.